src/lib.rs

```diff
@@ -121,33 +121,17 @@ fn return_to_relier(app: &AppConfig, redirect_uri: &str, params: &[(&str, &str)]
 /// The `broker_handler!` macro calls this on error. We don't use a `From`
 /// implementation, because this way we get app and request context, and we
 /// don't necessarily have to pass the error on to Iron.
+///
+/// Based on the kind of error, we may log the error, hide the description
```

Comment:
Given the amount of restructuring here, this should probably be squashed into the previous commit that touched this code.